kinto/core/permission/postgresql/__init__.py

```diff
@@ -5,12 +5,15 @@
 
 from kinto.core.permission import PermissionBase
 from kinto.core.storage.postgresql.client import create_from_config
+from kinto.core.storage.postgresql.migrator import Migrator
 
 
 logger = logging.getLogger(__name__)
 
+HERE = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Why the `os.path.abspath`? `os.path.dirname` should always give a valid directory path.

Reply:
What I mean is; If there's a strong reason for using `os.path.abspath` it should be a code comment that explains why. If there isn't a reason, it can be deleted and only use `os.path.dirname(__file__)`.

Reply:
Apparently as of Python 3.4, `__file__` is almost always an absolute path. I have updated the uses of it in the codebase.